### ISSUE TITLE
after trt optimization BatchNorm2d between conv2d and relu will be fused, but got wrong result.

### ISSUE THREAD

**Issue by ys0232** [onnx/onnx-tensorrt#594]
```

class myModel(nn.Module):
      def __init__(self):
            super(myModel,self).__init__()
            n_feats = 64
           self.conv1 = nn.Conv2d(3, n_feats,3)
           self.bn1 = nn.BatchNorm2d(n_feats)
           self.relu = nn.ReLU(inplace=True)
     def forward(self,x):
           x = self.conv1(x)
           x = self.bn1(x)
           x = self.relu(x)
        return x
```

- pytorch1.7.0 or pytorch1.4.0
- trt7.0 or trt7.1

pytorch1.7.0 will remove the BatchNorm2d, pytorch1.4.0 still keep BatchNorm2d.
trt 7.0 and trt7.1 will fuse conv2d weight with scales from batchNorm2d, and fuse with relu.
after trt optimization, only one layer in the network, and the result of this optimized network is totally different from the original one running on pytorch.

can anyone help to explain that?

**Comment by kevinch-nv** [comment]
Thanks for the report. Can you provide us a script and the generated ONNX model to help reproduce? Do you see the same error with TRT 7.2?

**Comment by kevinch-nv** [comment]
Closing this due to inactivity - if you're still having an issue with the latest version of TRT feel free to open a new issue.
